recipes/yake/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "yake" %}
+{% set version = "0.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c67afd5f1313bc521f43709ff2efd67c12073b6f448bdf2f2de59b8566369f6a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Looks like you need an `entry_points` item because that's in the `setup.py`:
https://github.com/LIAAD/yake/blob/2d025e3abf85ac04307cb50b8faa7249861a632c/setup.py#L66-L68

Reply:
Thanks for noticing it. It is added now.